subsys/net/lib/nrf_cloud/src/nrf_cloud_agps.c

```diff
@@ -14,10 +14,13 @@
 #include <cJSON_os.h>
 #include <modem/modem_info.h>
 #include <net/nrf_cloud_agps.h>
+#if defined(CONFIG_NRF_CLOUD_CELL_POS)
```

Comment:
might as well create a new c file too, `nrf_cloud_cell_pos.c` and move everything in there that is inside `#if defined(CONFIG_NRF_CLOUD_CELL_POS)` blocks

Reply:
This is the difficult part, as AGPS and Cell_Pos share common function calls.
I'll create a new source file for cell_pos and check how to separate them completely.